jenkins/deploy-clio-server.sh

```diff
@@ -170,6 +168,11 @@ done
 if [[ "$STATUS" == "OK" && "$REPORTED_VERSION" == "$DOCKER_TAG" ]]; then
   # rm /app.old.old
   ${SSHCMD} "sudo rm -rf ${APP_DIR}.old.old"
+
+  # tag the current commit with the name of the environment that was just deployed to
+  # this is a floating tag, so we have to use -f
+  git tag -f "$ENV"
```

Comment:
We can't push here because this script runs with Jenkis' SSH key for GCE, not GitHub, so we have to use the Git publisher plugin to push as a follow-up step. I'll add a comment here to explain it.